modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -0,0 +1,49 @@
+:experimental:
+= Migrating from CoreOS Container Linux (CL) to Fedora CoreOS (FCOS)
+
+Because Container Linux is being folded into the FCOS project, it will reach its End-of-Life sometime in 2020. This page documents differences between CL and FCOS to ease the transition to FCOS.
```

Comment:
Not quite "folded in".  `Fedora CoreOS is the successor to CoreOS Container Linux, which will reach its End of Life in mid-2020.`?